src/Components/SearchBar.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
```

Comment:
File name starting with capital letter

Reply:
https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76